check_raid.pl

```diff
@@ -559,15 +558,41 @@ sub program_names {
 
 sub commands {
 	{
-		'status' => ['-|', '@CMD'],
+		'status' => [>&2', '@CMD'],
```

Comment:
On 16.02.2015 15:37, noblemtw wrote:
> I think ‘>&2’ is actually correct, it’s a special op used by the 
> command function, 

also, >&2 special op will switch to using STDERR instead of STDOUT, i.e 
it will not merge two streams, but switch to stderr.

and by your comments you are suggesting it really redirects and as 
result check_raid sees both streams merged?

-- 
glen